### DISCUSSION TITLE
Comments section for https://docs.digit.org/

### DISCUSSION THREAD

**Discussion by vipinvijay1986** [egovernments/Digit-Core#29] [Announcements]
Hi Team, 
If I have some query related to https://docs.digit.org/ pages then where should I post it. As such I could not find comments section / QA link on the digit-docs website.

**Comment by eGovMan** [comment]
Just post them here. Provide the link to the document you are referring to.

**Reply by vipinvijay1986** [reply]
@eGovMan actually it's ageneral point.. Neil informed me about the link so I was able to post my queries if someone new to docs.digit then they do not know about this discussion board.

**Comment by eGovMan** [comment]
Yes Vipin. We are actively working on making it easier for community members to engage and give feedback. The redesign of the new sites are under way and will share with you shortly to get your feedback.
